tests/functional/cylc-cat-log/09-cat-running/flow.cylc

```diff
@@ -29,6 +29,10 @@
     [[cat-log]]
         script = """
             cylc__job__wait_cylc_message_started
+
+            # wait for job log retrieval to complete
+            cylc__job__poll_grep_workflow_log 'job-logs-retrieve.*ret_code.* 0$'
+
```

Comment:
Job log retrieval takes longer when PBS spooling is in play.

Make the test wait for retrieval to succeed to prevent erroneous failures.